services/director-v2/requirements/_base.in

```diff
@@ -29,6 +29,7 @@ networkx
 ordered-set
 orjson
 pydantic[dotenv]
+python-socketio
```

Comment:
we are slowly getting that library all over the place... nice ;(

Reply:
yes i agree with you. I will try to look for a solution for this sooner than later. In reality we only need a single function for this service when used like ANE is using it here.

Reply:
Yeah I know it's not ideal. We'll refactor them.